src/main/resources/application-prod.yml

```
@@ -1,31 +1,6 @@
     hibernate:
       ddl-auto: update
-    properties:
-      hibernate:
-        order_inserts: true
-        order_updates: true
-        jdbc:
-          batch_size: 50
-        dialect: org.hibernate.dialect.MySQLDialect
\ No newline at end of file
+  datasource:
+    url: jdbc:mysql://${DB_URL}:3306/${DB_NAME}?serverTimezone=Asia/Seoul
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)